### PR TITLE
add notice to camper news

### PR DESCRIPTION
**Pull request by QuincyLarson**

### CODE REVIEW

File: server/views/stories/news-nav.jade [hunk 1]
```diff
@@ -1,4 +1,14 @@
 .row
+    col-xs-12
+        h2 We are retiring Camper News in favor of our
```
Comment (BerkeleyTrue) [inline server/views/stories/news-nav.jade]: You're missing a dot in front of `col`

Comment (DavidCox1979) [inline server/views/stories/news-nav.jade]: Thanks for the help. Good looking out.

**Comment by QuincyLarson** [comment]
@DavidCox1979 @BerkeleyTrue Thanks for catching this! I've fixed this and squashed my commit.

**Comment by SaintPeter** [comment]
:+1:

**Pull request merged**